readthedocs/restapi/views/integrations.py

```diff
@@ -83,12 +88,18 @@ def finalize_response(self, req, *args, **kwargs):
 
     def get_data(self):
         """Normalize posted data."""
-        return normalize_request_payload(self.request)
+        return normalize_request_payload(
+            self.body,
+            self.request.content_type
+        )
 
     def handle_webhook(self):
         """Handle webhook payload."""
         raise NotImplementedError
 
+    def validate_payload(self):
```

Comment:
Docstring missing here.

Also, I think `is_payload_valid` would be a better name.